projects/cm_mcu/ZynqMonTask.c

```diff
@@ -43,27 +44,37 @@
 // Register List
 // See Google Docs, 'CM uC Sensor register map'
 
-#define SENSOR_MESSAGE_START_OF_FRAME_NIB 2
-#define SENSOR_MESSAGE_DATA_FRAME_NIB     0
-#define SENSOR_MESSAGE_HEADER_OFFSET      6
-#define SENSOR_SIX_BITS                   0x3F
+#define SENSOR_MESSAGE_START_OF_FRAME_NIB    2
+#define SENSOR_MESSAGE_START_OF_FRAME_NIB_V2 3
+#define RESERVED_DATA                        0x9 // 0b1001
```

Comment:
fwiw, this just a dummy value. We don't need to put this in

Reply:
do you mean typing them directly to the code?